### PR TITLE
Hide Specific References table when it's empty

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When there's no specific references of product, we should not show the table of these references in FO |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1552 |
| How to test? | Add a new product with no specific reference and visualize it |

### CODE REVIEW

File: src/Core/Product/ProductPresenter.php [hunk 1]
```diff
@@ -592,6 +595,7 @@ public function present(
                 $presentedProduct
             );
         }
+        /*dump($presentedProduct);*/
```
Comment (aleeks) [inline src/Core/Product/ProductPresenter.php]: Hello @fatmaBouchekoua,
Can you delete this line please ?!

Comment (fatmaBouchekoua) [inline src/Core/Product/ProductPresenter.php]: Hello, 
Done Thanks ;)

**Review by aleeks — APPROVED** [review]

**Comment by dSkrbic** [comment]
capture

**Comment by thierrymarianne** [comment]
Thanks!

**Pull request merged**